sharding/nodesCoordinator/numberOfShardsComputer.go

```diff
@@ -0,0 +1,24 @@
+package nodesCoordinator
+
+type numberOfShardsWithMetaComputer struct {
+}
+
+// NewNumberOfShardsWithMetaComputer creates a new instance of the number of shards computer with meta
+func NewNumberOfShardsWithMetaComputer() *numberOfShardsWithMetaComputer {
```

Comment:
I don't think these need to be exported, since they are only used internally

Reply:
fixed